tdrs-backend/tdpservice/users/management/commands/generate_dev_user.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.core.management import BaseCommand
+
+User = get_user_model()
+
+email = "dev@test.com"
+pswd = "pass"
+first = "Jon"
+last = "Tester"
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        try:
+            user = User.objects.get(username=email)
+            print(f"Found {vars(user)}")
+        except User.DoesNotExist:
+            group = Group.objects.get(name="Developer")
+            user = User.objects.create(username=email,
```

Comment:
can the user be given `is_superuser=True` and `is_staff=True`? this will open up the "Admin" tab

Reply:
hi Jan, FYI - have been talking with Eric; there are a few branches outstanding now; we're now thinking to reopen this off of 1858-kibana-ofa-access